.github/ISSUE_TEMPLATE/font.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Font
+about: Issues and improvements for the font
+---
+
+<!-- Before opening a new issue search for duplicate or closed issues -->
+
+
+### Kind of issue <!-- Change the one that applies to `[x]`  -->
+
+- [ ] Improvement
+- [ ] Bug
+- [ ] Other, namely:
+
+### Description
+
+
```

Comment:
This line should be removed?

Reply:
I just copied the template form the templates of the main repository, the reason for it being there is that, in my opinion, it provides some additional space for the user to click between the title and comment so that they can start typing.

Reply:
Ahm, okay then :+1: